doc/QCDB.md

```diff
@@ -0,0 +1,198 @@
+
+QCDB
+---
+
+<!--TOC generated with https://github.com/ekalinin/github-markdown-toc-->
+<!--./gh-md-toc --insert --no-backup --hide-footer --indent 3 QualityControl/doc/Advanced.md -->
+<!--ts-->
+   * [QCDB vs CCDB](#qcdb-vs-ccdb)
+   * [Details on the data storage format in the QCDB](#details-on-the-data-storage-format-in-the-qcdb)
+   * [Custom metadata for QC objects in the QCDB](#custom-metadata-for-qc-objects-in-the-qcdb)
+   * [Instructions to move an object in the QCDB](#instructions-to-move-an-object-in-the-qcdb)
+   * [Accessing objects in CCDB](#accessing-objects-in-ccdb)
+   * [Access GRP objects with GRP Geom Helper](#access-grp-objects-with-grp-geom-helper)
+   * [Global Tracking Data Request helper](#global-tracking-data-request-helper)
+   * [Local CCDB setup](#local-ccdb-setup)
+<!--te-->
+
+## QCDB vs CCDB
+
+The MonitorObjects generated by Quality Control are stored in a dedicated repository called **QCDB**.
+The run conditions, on the other hand, are located in another, separate database, called **CCDB**.
+
+Both are based on a technology called _CCDB_ which does not help with the confusion...
+
+## Details on the data storage format in the QCDB
+
+Each MonitorObject is stored as a TFile in the QCDB.
+It is therefore possible to easily open it with ROOT when loaded with alienv. It also seamlessly supports class schema evolution.
+
+The MonitorObjects are stored at a path which is enforced by the qc framework : `/qc/<detector code>/MO/<task name>/object/name`
+Note that the name of the object can contain slashes (`/`) in order to build a sub-tree visible in the GUI.
+The detector name and the taskname are set in the config file :
+
+```json
+"tasks": {
+  "QcTask": {       <---------- task name
+    "active": "true",
+    "className": "o2::quality_control_modules::skeleton::SkeletonTask",
+    "moduleName": "QcSkeleton",
+    "detectorName": "TST",       <---------- detector name
+```
+
+The quality is stored as a CCDB metadata of the object.
+
+## Custom metadata for QC objects in the QCDB
+
+One can add custom metadata on the QC objects produced in a QC task.
+Simply call `ObjectsManager::addMetadata(...)`, like in
+
+```
+  // add a metadata on histogram mHistogram, key is "custom" and value "34"
+  getObjectsManager()->addMetadata(mHistogram->GetName(), "custom", "34");
+```
+
+This metadata will end up in the _QCDB_.
+
+It is also possible to add or update metadata of a MonitorObject directly:
+
+```
+  MonitorObject* mo = getMonitorObject(objectName);
+  mo->addOrUpdateMetadata(key, value);
+```
+
+## Instructions to move an object in the QCDB
+
+The script `o2-qc-repo-move-objects` lets the user move an object, and thus all the versions attached to it. E.g.:
+
+```
+python3 o2-qc-repo-move-objects --url http://ccdb-test.cern.ch:8080 --path qc/TST/MO/Bob --new-path qc/TST/MO/Bob2 --log-level 10 
+```
```

Comment:
do you feel confident advertizing this to users? If not, it could be moved to DeveloperTips.md

Reply:
yes, indeed. 
there was also something about `truncate` that I removed. 😂